### ISSUE TITLE
Special chars in `release` var break the html search

### ISSUE THREAD

**Issue by rakhimov** [sphinx-doc/sphinx#3410]
This is silly but it was hard bug to track.
Automatically getting the release version with git and forgetting to trim the output
results in subtle failure of the html search.
```python
release = check_output(['git', 'describe', '--tags', commit_hash]).decode()
```
Testing with a raw string with a newline character reveals the problem.

P.S. Probably this is documented somewhere, but I couldn't find.

**Comment by tk0miya** [comment]
I'd like to reproduce at my local, but I don't have the implementation of`check_output()` function.
Could you show me reproducible example?

**Comment by rakhimov** [comment]
@tk0miya try

```python
release = '1.0\n'
```

**Comment by tk0miya** [comment]
Thanks, I'll take a look

**Comment by rakhimov** [comment]
The ``check_output`` is from ``subprocessor``,
but the implementation shouldn't be the cause.
The output seems to be new-line terminate from check_output.

**Comment by tk0miya** [comment]
I reproduced.
I found warning from Chrome: `Uncaught SyntaxError: Invalid or unexpected token`.

With default theme, Sphinx generates following script. But it is invalid.
```
    <script type="text/javascript">
      var DOCUMENTATION_OPTIONS = {
        URL_ROOT:    './',
        VERSION:     '1.0
',
        COLLAPSE_INDEX: false,
        FILE_SUFFIX: '.html',
        HAS_SOURCE:  true,
        SOURCELINK_SUFFIX: '.txt'
      };
    </script>
```

We have to replace the return-code to any other character (like white space).

**Comment by tk0miya** [comment]
Fixed at 387588a.
Thank you for reporting
